carbon-design-system/carbon
# feat(filterable-multiselect): add onMenuChange event

Closes #6434 

This is very similar to #7288, but instead adds the `onMenuChange` event functionality to the [`FilterableMultiSelect`](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/MultiSelect/FilterableMultiSelect.js) component.

#### Changelog

**New**

- `onMenuChange` event on filterable multiselect

**Changed**


**Removed**


#### Testing / Reviewing

- Start storybook
- Open filterable multiselect story
- Open the multiselect menu
- See onMenuChange: (1) [true]
- Close the multiselect menu
- See onMenuChange: (1) [false]
